### DISCUSSION TITLE
Can i move app directory under another folder ?

### DISCUSSION THREAD

**Discussion by indrazm** [vercel/next.js#67999] [App Router]
Hey Anyone.

I just wondering, can i move app directory under nother folder ?
Let say :

```ts
- src
-- presentation
--- app
---- page.tsx
---- layout.tsx
```

Thanks!

**Comment by romhenri** [comment]
I think no, because the tool will search "app" in src. Moreover, if this possible, would cause problem becausa you can create a app inside the app and the main will be of src direct parent.

**Comment by indrazm** [comment]
I see. 
I mean if it's flexible, it would be great.
even with adding configuration in next.config. just like "

```js
module.export = {
   appDirPath: './src/presentation/app'
}
```

Alright. Thanks bro!

**Comment by icyJoseph** [comment]
Nope, it'll look directly for `src/app`, or `<root-of-project>/app`.

What is it that you are expecting to gain? Maybe there is another way to do that?
